content/people/levorato.md

```diff
@@ -32,6 +32,7 @@ address: |
 bibliography:
 - title: 2018
   items:
+  - callegaro-levorato-2018-globecom
```

Comment:
Thank you for that. The motivation for adding the reference to a new
paper coauthored with Marco to the bibliography on his personal page
is that his references list (at least, for now) should be the copy of
the lab references list used to generate the global Publications page.

At the moment, the easiest way to make sure that the identity of the
two lists is maintained is to always add entries coauthored with Marco
into both places as a part of a single pull request.

It will soon be laid out in the checklist instructions for adding
a new bibliography entry in the README file. I am almost done
with those.